### PR TITLE
🎉 feat: Migrate to Lightning Web Runtime

### PR DESCRIPTION
**Pull request by adityanaag3**
### What does this PR do?
Migrates the app from deprecated lwc-services to Lightning Web Runtime

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,11 +59,11 @@
             "eslint"
         ],
         "**/src/**": [
-            "sfdx-lwc-jest --skipApiVersionCheck -- --bail --findRelatedTests"
+            "jest"
```
Comment (pozil) [inline package.json:62]: Any reason why you removed the extra `--bail --findRelatedTests` flags from the command?
I would rather keep them.

**Review by pozil — APPROVED** [review]
Great job on the migration @adityanaag3. I just have one open question for you before you merge.

File: package.json (hunk 1 above)
Comment (adityanaag3) [reply package.json]: Good catch. Added the flags.

File: src/layouts/index.html [hunk 2]
```diff
@@ -140,6 +140,6 @@ <h1 class="title">Lightning Web Components Recipes</h1>
             </ul>
         </header>
 
-        <div id="main"></div>
+        <div id="main">{{{body}}} {{{lwr_resources}}}</div>
```
Comment (muenzpraeger) [inline src/layouts/index.html:143]: `{{{ lwr_resources }}}` should be a child element to the `body`.

Comment (adityanaag3) [reply src/layouts/index.html:143]: Fixed it in https://github.com/trailheadapps/lwc-recipes-oss/pull/274

Comment (ryanwhu) [reply src/layouts/index.html:143]: With the new LWR, how should we import "@lwc/synthetic-shadow"?

Comment (adityanaag3) [reply src/layouts/index.html:143]: In lwr.config.json, you'll have to add `"bootstrap": { "syntheticShadow": true}` as an option in the route for which you want synthetic shadow. For example - 
```
"routes": [
        {
            "id": "home",
            "path": "/",
            "rootComponent": "ui/app",
            "layoutTemplate": "$layoutsDir/index.html"
            "bootstrap": {
                "syntheticShadow": true
            }
        }
    ]
```

**Pull request merged**